src/main/docker/Dockerfile

```diff
@@ -17,7 +17,7 @@
 # Docker registry
 ARG DOCKER_HUB_PUBLIC=docker.io
 
-FROM ${DOCKER_HUB_PUBLIC}/cafapi/buildenv-dotnet:1.0.0
+FROM ${DOCKER_HUB_PUBLIC}/cafapi/prereleases:buildenv-dotnet-1.1.0-SNAPSHOT
```

Comment:
Why are you using 1.1.0-SNAPSHOT when Dermot suggested `cafapi/prereleases:buildenv-jdk8-2.0.0-SNAPSHOT`?

Reply:
That's ok.  `buildenv-dotnet-1.1.0-SNAPSHOT` is layered on top of `buildenv-jdk8-2.0.0-SNAPSHOT` (since the [CAFapi/buildenv-dotnet-image#4](https://github.com/CAFapi/buildenv-dotnet-image/pull/4/files) changes).